eya_def_tools/eya_def_tools/data_models/spatial.py

```diff
@@ -6,34 +6,17 @@
 
 from eya_def_tools.data_models.base_model import EyaDefBaseModel
 
-x_field: float = pdt.Field(
-    ...,
-    description="Location x-coordinate (typically easting).",
-    examples=[419665.0],
-)
-
-y_field: float = pdt.Field(
-    ...,
-    description="Location y-coordinate (typically northing).",
-    examples=[6195240.0],
-)
-
 
 class Location(EyaDefBaseModel):
     """Specification of a horizontal location in space."""
 
-    x: float = x_field
-    y: float = y_field
-
-
-class IdLocation(EyaDefBaseModel):
```

Comment:
The EYA DEF data model for a reference meteorological dataset allowed for specifying multiple locations for a dataset. The IEA Wind Task 43 WRA Data Model does not support that. Looking at it now, I think it is better to keep it simpler and avoid the added complexity of having multiple locations for the same dataset. The user just needs to specify one dataset for each location. Now that we support the IEA Wind Task 43 WRA Data Model format only, we do not need represent different locations with different location IDs anymore.